src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -131,9 +131,13 @@ class ModuleExecutor(
         onQueryNotAtResultEnd: () -> Unit,
         newPostedCommentCache: Cache<MutableSet<String>>
     ): List<Issue> {
-        val projects = (config[moduleConfig.projects] ?: config[Arisa.Issues.projects])
-        val resolutions = config[moduleConfig.resolutions].map(String::toLowerCase)
+        val projects = config[moduleConfig.projects] ?: config[Arisa.Issues.projects]
```

Comment:
I really wish we could make this entire class more testable, but out of scope for this PR

Reply:
This class probably needs to be rewritten anyway because of #527